lib/steps/fetch_step.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/Originate/git-town/lib/script"
 )
 
+var fetched bool
```

Comment:
Ideally we do this without mutable state in this file. Maybe filter out repeated `FetchStep` instances from the steplist? This can be done later.